### PR TITLE
Fix: mono migration victor

### PR DESCRIPTION
**Pull request by victorgrubio**

### CODE REVIEW

File: udagram-api-user/src/aws.ts [hunk 1]
```diff
@@ -3,8 +3,10 @@ import {config} from './config/config';
 
 
 // Configure AWS
-const credentials = new AWS.SharedIniFileCredentials({profile: config.aws_profile});
-AWS.config.credentials = credentials;
+AWS.config.update({
+  accessKeyId: process.env.AWS_ACCESS_KEY_ID,
```
Comment (victorgrubio) [inline udagram-api-user/src/aws.ts:7]: For the sake of coherece, put this into config, and the use the config value as we do in other places. The change is fine :)

File: udagram-api-feed/src/aws.ts [hunk 2]
```diff
@@ -3,8 +3,10 @@ import {config} from './config/config';
 
 
 // Configure AWS
-const credentials = new AWS.SharedIniFileCredentials({profile: config.aws_profile});
-AWS.config.credentials = credentials;
+AWS.config.update({
```
Comment (victorgrubio) [inline udagram-api-feed/src/aws.ts:6]: For the sake of coherece, put this into config, and the use the config value as we do in other places. The change is fine :)

**Review by victorgrubio — COMMENTED** [review]
Change the env to config please 👍🏻

File: udagram-api-user/src/aws.ts (hunk 1 above)
Comment (kovaj024) [reply udagram-api-user/src/aws.ts:7]: Understood

**Pull request merged**